src/apps/users/api/views.py

```diff
@@ -92,8 +91,7 @@ def verify(self, request, token):
 
     @action(detail=True, methods=["PATCH"])
     def change_password(self, request, *args, **kwargs):
-        serializer = self.get_serializer(data=request.data)
-        user = get_object_or_404(User, username=request.user)
-        message, is_valid = user_change_password(serializer, user)
-        status_code = status.HTTP_200_OK if is_valid else status.HTTP_400_BAD_REQUEST
-        return Response(message, status=status_code)
+        serializer = self.get_serializer(data=request.data, instance=request.user)
+        if serializer.is_valid(raise_exception=True):
+            serializer.save()
```

Comment:
NO NEEEED to use `if` here. if it is not valid the ValidationError gonna be raised:)
```python
serializer.is_valid(raise_exception=True)
serializer.save()
return Response("...")
```